exercises/toRoman/toRoman.js

```diff
@@ -15,14 +15,47 @@
  * @returns {string} The input integer's Roman numeral representation
  */
 function toRoman(num) {
-  // This is your job. :)
+  let ones = ["I","IV","V","VI","VII","VIII","IX","X"];
+  let tens = ["X","L","C"];
+  let hundereds = ["D","M"];
+  let n = num.toString();
+  let ROM = [];
+  ROM = n.split("");
+  console.log(ROM);
```

Comment:
Remove debugging output like this (or keep it hidden behind some kind of debug flag). At the very least, make the debugging messages more useful so I can get a sense of why you're interested in printing out these values.